src/app/layout/redirect-status/status/status.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="blank-slate-pf">
+  <div class="rh-branding-logo">
+    <img class="width-100"
+      src="../../../../assets/images/Logotype_RH_OpenShift-io_RGB_RedGray.png">
```

Comment:
I seem to recall that there are aliased paths for things like this, but I don't know if Webpack is configured so that those paths are usable from within HTML. Could you double check that, so these paths can look like `~assets/images/foo` and avoid all the relative pathing `../../` parts?

Reply:
I can't seem to find anywhere in `fabric8-ui` that uses links like that. Everywhere it seems to be relative path so i don't think we have webpack setup for that. Might be worth looking into it though.

Reply:
There's some configuration for testing utils at least. We definitely have specs that do `import { createMock } from 'testing/mock'` for example. If there's nothing similar for assets then I'd like to see it added, but that's a separate concern from this PR.